### PR TITLE
test: fix remaining DaemonSet induced flakes

### PR DESCRIPTION
**Pull request by jmdeal**
Fixes #N/A 

**Description**

**How was this change tested?**

**Does this change impact docs?**
- [ ] Yes, PR includes docs updates 
- [ ] Yes, issue opened: # 
- [x] No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by jmdeal — COMMENTED** [review]
/karpenter snapshot

**Review by jmdeal — COMMENTED** [review]
/karpenter snapshot

**Review by jmdeal — COMMENTED** [review]
/karpenter snapshot

File: .github/workflows/e2e-upgrade.yaml [hunk 1]
```diff
@@ -108,7 +108,7 @@ jobs:
           region: ${{ inputs.region }}
           cluster_name: ${{ steps.generate-cluster-name.outputs.CLUSTER_NAME }}
           k8s_version: ${{ inputs.k8s_version }}
-          eksctl_version: v0.169.0
+          eksctl_version: v0.175.0
```
Comment (engedaam) [inline .github/workflows/e2e-upgrade.yaml:111]: Just wondering, is there a reason for this change?

**Review by engedaam — APPROVED** [review]
LGTM 🚀

**Pull request merged**